scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -49,25 +52,28 @@ terms and associated correlation parameters in the RE structure.
 ## Model fit
 
 ```julia
-const HC = HelmertCoding();
-const contrasts = Dict(:F => HC, :P => HC, :Q => HC, :lQ => HC, :lT => HC);
+contrasts = merge(
+    Dict(nm => HelmertCoding() for nm in (:F, :P, :Q, :lQ, :lT)),
+    Dict(nm => Grouping() for nm in (:Subj, :Item)),
+);
 
 m1form = @formula (-1000/rt) ~ 1+F*P*Q*lQ*lT +
                               (1+F+P+Q+lQ+lT | Subj) +
                               (1+P+Q+lQ+lT | Item);
-cmplxLMM = @time fit(MixedModel, m1form, dat, contrasts=contrasts);
+cmplxLMM = fit(MixedModel, m1form, dat, contrasts=contrasts);
+cmplxLMM.PCA
```

Comment:
For the sake of the Jupyter notebooks I think we should be in the habit of having something printed at the end of each code block.  Notice that `.PCA` gives the principal components analysis on the correlation scale.  @kliegl and @palday are better able to judge if that is preferred to the covariance scale.